modin/core/execution/ray/common/utils.py

```diff
@@ -126,6 +127,7 @@ def initialize_ray(
                 "object_store_memory": object_store_memory,
                 "_redis_password": redis_password,
                 "_memory": object_store_memory,
+                "resources": RayInitCustomResources.get(),
```

Comment:
Can we add a test for this case?

Reply:
What exactly would you like to test with this?

Reply:
We do not test the situation when this config is different from `None`.